src/applications/personalization/profile/mocks/endpoints/address.js

```diff
@@ -107,7 +107,7 @@ const addressValidation = {
         zipCode: '94115',
       },
       addressMetaData: {
-        confidenceScore: 0,
+        confidenceScore: 88,
```

Comment:
I see this is a mock, but what's the purpose for changing this?

Reply:
I was testing with different confidenceScore to see if UI changes somehow to better understand how the address validation worked @nickjg231